samples/rest/BingSpellCheckV7.py

```diff
@@ -6,23 +6,31 @@
 from pprint import pprint
 import requests
 import urllib.parse
+from dotenv import load_dotenv
+
+# Load the environment variables from .env file
+load_dotenv()
 
 '''
 This sample uses the Bing Spell Check API to check the spelling of query words 
 and then suggests corrections with a scored confidence.
 Bing Spell Check API: https://docs.microsoft.com/en-us/bing/search-apis/bing-spell-check/overview
 '''
+AUTH_HEADER_NAME='Ocp-Apim-Subscription-Key'
+SUBSCRIPTION_KEY_ENV_VAR_NAME='BING_SEARCH_V7_SPELL_CHECK_SUBSCRIPTION_KEY'
 
-# Add your Bing Spell Check subscription key and endpoint to your environment variables.
-key = os.environ['BING_SPELL_CHECK_SUBSCRIPTION_KEY']
-endpoint = os.environ['BING_SPELL_CHECK_ENDPOINT'] + '/bing/v7.0/spellcheck'
+# Add your Bing Spell Check subscription key to your environment variables / .env file
+subscription_key = os.environ.get(SUBSCRIPTION_KEY_ENV_VAR_NAME)
+if subscription_key is None:
+    raise(RuntimeError(f'Please define the {SUBSCRIPTION_KEY_ENV_VAR_NAME} environment variable'))
 
 # Query you want spell-checked. 
-query = 'Hollo, wrld!'
+query = '''when i went two the houze i heared they'r'e voice and they're srcreams. I walk their and told: "helo fren"'''
```

Comment:
When i went two the house i heard their voices and their screams. I walked there and said: "hello fren"

Reply:
hello fren, howiz itgoin

Reply:
Whatchulookinat :)